eng/helix/helix.proj

```diff
@@ -21,11 +22,12 @@
     <IsExternal>true</IsExternal>
     <SkipInvalidConfigurations>true</SkipInvalidConfigurations>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
+    <HelixAccessToken Condition="'$(UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(ContinuousIntegrationBuild)' == 'true' ">
     <HelixType>ci</HelixType>
-    <Creator>aspnetcore</Creator>
+    <Creator Condition="'$(UseHelixOpenQueues)' == 'true'">aspnetcore</Creator>
```

Comment:
Why is this conditional? Looks like it will leave the `Creator` metadata out in official builds.

Reply:
The internal queues reject jobs that have creator specified, so yes this is intentional as creator isn't allowed for some reason, probably api token is the creator

Reply:
Please add a comment so someone doesn't later "fix" not assigning this property in internal builds.